### PR TITLE
[PHEE-666] Reliability Testing

### PR DESCRIPTION
**Pull request by Anover000**
[PHEE-666 Reliability Testing](https://fynarfin.atlassian.net/browse/PHEE-666?atlOrigin=eyJpIjoiZTMxNzYyZGZkNjIxNGNjN2IwNDBhNTdjYzYwNmQyNzEiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!
- [ ] Followed the PR title naming convention mentioned above.

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Created/updated unit or integration tests for verifying the changes made.

- [ ] Added required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

File: helm/ph-ee-engine/values.yaml [hunk 1]
```diff
@@ -85,7 +85,7 @@ camunda-platform:
       - name: init-ph-ee-kafka-exporter
         image: busybox:1.28
         command: ['/bin/sh', '-c']
-        args: ['wget -O /exporters/ph-ee-kafka-exporter.jar "https://fynarfin.io/images/exporter-1.1.1-SNAPSHOT.jar"; ls -al /exporters/']
+        args: ['wget -O /exporters/ph-ee-kafka-exporter.jar "https://fynarfin.io/images/exporter-3.0.0-SNAPSHOT.jar"; ls -al /exporters/']
```
Comment (fynmanoj) [inline helm/ph-ee-engine/values.yaml]: if the backward compatibility is not broken, increase the minor version

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: helm/ph-ee-engine/values.yaml (hunk 1 above)
Comment (Anover000) [reply helm/ph-ee-engine/values.yaml]: Updated the version.

**Review by fynmanoj — APPROVED** [review]

**Review by fynmanoj — APPROVED** [review]

File: helm/ph-ee-engine/connector-channel/templates/service.yaml [hunk 2]
```diff
@@ -22,5 +22,5 @@ spec:
   selector:
     app: ph-ee-connector-channel
   sessionAffinity: None
-  type: ClusterIP
+  type: LoadBalancer
```
Comment (fynmanoj) [inline helm/ph-ee-engine/connector-channel/templates/service.yaml]: reverse this as it is not needed now,

**Review by fynmanoj — CHANGES_REQUESTED** [review]

**Comment by fynmanoj** [comment]
update PR title

File: helm/ph-ee-engine/connector-channel/templates/service.yaml (hunk 2 above)
Comment (Anover000) [reply helm/ph-ee-engine/connector-channel/templates/service.yaml]: Done.

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**